src/net/sourceforge/kolmafia/textui/command/PingCommand.java

```diff
@@ -38,7 +38,7 @@ public void run(final String cmd, String parameters) {
         }
       }
       if (split.length > 2) {
-        verbose = split[2].equals("true");
+        verbose = split[2].equals("true") || split[2].equals("true");
```

Comment:
think you meant .equals("verbose") in one of these.

Reply:
Sigh.